arch/arm/core/fault_s.S

```diff
@@ -127,6 +135,7 @@ _s_stack_frame_msp:
 	mrs r0, MSP
 _s_stack_frame_endif:
 #endif /* CONFIG_ARM_SECURE_FIRMWARE || CONFIG_ARM_NONSECURE_FIRMWARE */
+#elif defined(CONFIG_ARMV7_R)
```

Comment:
MAybe an inline comment justifying why this is empty?

Reply:
fixed

Reply:
ok